packages/http/src/resources/HotspotRewards.ts

```diff
@@ -1,11 +1,17 @@
 import type Client from '../Client'
-import { HotspotReward, HotspotRewardSum, HTTPHotspotReward } from '../models/HotspotReward'
+import {
+  HotspotReward,
+  HotspotRewardSum,
+  HTTPHotspotReward,
+  HTTPHotspotRewardSum,
+} from '../models/HotspotReward'
 import ResourceList from '../ResourceList'
 
 interface ListRewardsParams {
-  minTime?: Date
-  maxTime?: Date
+  minTime?: Date | string // Date or natural format eg: '-30 day' '-1 week' '-5 hour'
```

Comment:
it looks like there might be a way to define the natural language param as a template literal type: https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html

Reply:
Nice, added a `NaturalDate` type